packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -71,6 +71,10 @@ const ProductTile = (props) => {
     // name as `productName`. ProductList provides a localized name as `productName` and does not
     // use the `name` property.
     const localizedProductName = product.name ?? product.productName
+    const strikePrice =
+        product.c_extend &&
+        product.c_extend?.priceInfo?.originalPrice?.value != price &&
+        product.c_extend?.priceInfo?.originalPrice?.value
```

Comment:
Small note that this could be 

```
const strikePrice =
        product?.c_extend?.priceInfo?.originalPrice?.value != price &&
        product?.c_extend?.priceInfo?.originalPrice?.value
```

Reply:
Fair point, I changed it accordingly.